### ISSUE TITLE
bootstrap glyphicon sort chevrons not showing

### ISSUE THREAD

**Issue by Akibalogh** [MoonStorm/trNgGrid#112]
when user clicks a sort button, the sort chevron is not showing.

instead of the css being "glyphicon-chevron-up" it should be "glyphicon glyphicon-chevron-up"
the same goes for glyphicon-chevron-down

**Comment by MoonStorm** [comment]
Do you guys see this on the demo site as well? Or here http://jsfiddle.net/MoonStorm/pkuca2f8/ ?

**Comment by Akibalogh** [comment]
seems to be working correctly on the demo site and on the fiddle (using OSX 10.10, Chrome v39)

**Comment by peterski74** [comment]
Yes I can see the chevron on the demo site http://jsfiddle.net/MoonStorm/pkuca2f8/, I'll try the latest release code maybe it has been fixed now.

**Comment by gaiazov** [comment]
It looks like the 'glyphicon' class gets removed because it is used in both 'normal' and 'reverse' bindings. For me, it happens when using angular 1.2.16 and went away after I upgraded to 1.2.28. Looking at the changelog there were some fixes in later angular releases to address overlapping classes in ng-class (e.g. https://github.com/angular/angular.js/commit/c9677920d462046710fc72ca422ab7400f551d2e)

Here's a fiddle, all I did was change to angular 1.2.16 - http://jsfiddle.net/MoonStorm/pkuca2f8/
